### PR TITLE
wasm, core: actions: add admin auth builder & open orders hook

### PR DESCRIPTION
**Pull request by akirillo**
This PR adds a `build_admin_headers` helper in Rust for creating admin authentication headers for a request, and uses it in a new hook to fetch open orders from a relayer to which one has admin access.

I also cleaned up all the warnings in the `wasm` crate

### CODE REVIEW

File: packages/react/src/exports/index.ts [hunk 1]
```diff
@@ -138,6 +138,12 @@ export {
 
 export { useWallet } from '../hooks/useWallet.js'
 
+export {
+  useOpenOrders,
+  type UseOpenOrdersParameters,
+  type UseOpenOrdersReturnType,
+} from '../hooks/useOpenOrders.js'
+
```
Comment (sehyunc) [inline packages/react/src/exports/index.ts:146]: extremely small nit, can we alphabetize exports

File: wasm/Cargo.toml [hunk 2]
```diff
@@ -10,7 +10,7 @@ crate-type = ["cdylib"]
 [dependencies]
 getrandom = { version = "0.2", features = ["js"] }
 renegade-crypto = { git = "https://github.com/renegade-fi/renegade.git", default-features = false }
-sha2 = "0.10.8"
+sha2 = { version = "0.10.8", features = ["asm"] }
```
Comment (sehyunc) [inline wasm/Cargo.toml:13]: just making sure the feature flag is supposed to be `"asm"` and not `"wasm"`

**Review by sehyunc — APPROVED** [review]
LGTM, just small nits

File: wasm/Cargo.toml (hunk 2 above)
Comment (akirillo) [reply wasm/Cargo.toml:13]: yup, `"asm"` for "assembly"

File: packages/react/src/exports/index.ts (hunk 1 above)
Comment (akirillo) [reply packages/react/src/exports/index.ts:146]: 🫡

**Pull request merged**